package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "signal_generator",
+	"version": "0.0.1",
+
+	"author": "Murilo Mendonca Venancio",
+	"description": "Signal generator for learning basic Hyro concepts",
+
+	"dependencies": {
+		"hyro_api_channel": "*",
+		"hyro_utils": "*",
+		"hyro_msgs": "*"
```

Comment:
Missing dependencies.
Usually all required packages in CMakeLists.txt in `hyro_find_package(hyro_xxx REQUIRED)` should be in the package.json dependecies